tools/qontract_cli.py

```diff
@@ -8,11 +8,7 @@
 import re
 import sys
 from collections import defaultdict
-from datetime import (
```

Comment:
Can you check your `ruff` config? Looks like a bunch of imports got re-sorted around

Reply:
ugh my ruff version indeed got updated. I've reverted the non-relevant changes manually

Reply:
Thanks!